patches/core/ungoogled-chromium/fix-building-without-enabling-reporting.patch

```diff
@@ -22,6 +22,20 @@
        "net/reporting_service_proxy.cc",
        "net/reporting_service_proxy.h",
      ]
+--- a/content/public/common/content_switch_dependent_feature_overrides.cc
++++ b/content/public/common/content_switch_dependent_feature_overrides.cc
+@@ -47,9 +47,11 @@ GetSwitchDependentFeatureOverrides(const
+        std::cref(features::kDocumentPolicyNegotiation),
+        base::FeatureList::OVERRIDE_ENABLE_FEATURE},
+       {switches::kEnableExperimentalWebPlatformFeatures,
++#if BUILDFLAG(ENABLE_REPORTING)
+        std::cref(net::features::kDocumentReporting),
+        base::FeatureList::OVERRIDE_ENABLE_FEATURE},
+       {switches::kEnableExperimentalWebPlatformFeatures,
++#endif
```

Comment:
Is there a reason that you didn't include the first line of the struct you are trying to remove in the `#if ... #endif` and instead included the first line of the next struct in the array? While this will do what you intend, it looks a bit strange to me.

In my mind, the main reason you would not want to do this is that if first line of the next struct in the array is changed in a future version of Chromium to be different than the first line of the struct you are trying to remove, refreshing this patch will be awkward. It would be more clear if no portion of any other struct is removed.

Reply:
I'm not sure how I managed to get one line off, but you're right that it would cause problems as the entries get changed in the future.  I'll push an update in a bit.